proto/provenance/exchange/v1/events.proto

```diff
@@ -77,6 +77,30 @@ message EventOrderExternalIDUpdated {
   string external_id = 3;
 }
 
+// EventFundsCommitted is an event emitted when funds are committed to a market.
```

Comment:
Would it make sense to have an event for settlement?

Reply:
I thought about that, but decided that there wasn't anything extra that I'd be putting in there that isn't covered by the generic msg event automatically added.

Reply:
If you are listening for settlement, then what you are listening for?

Reply:
I can't remember the specifics, but it's the one that the SDK emits somewhere indicating the msg type being processed.